oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileStoreUtils.java

```diff
@@ -30,46 +32,52 @@
 import java.io.OutputStream;
 import java.io.OutputStreamWriter;
 import java.util.List;
-import java.util.zip.Deflater;
-import java.util.zip.GZIPInputStream;
-import java.util.zip.GZIPOutputStream;
-
-import static com.google.common.base.Charsets.UTF_8;
 
+/**
+ * This class provides common utility functions for building FlatFileStore.
+ */
 class FlatFileStoreUtils {
 
+    /**
+     * This function by default uses GNU zip as compression algorithm for backward compatibility.
```

Comment:
```suggestion
     * This function by default uses gzip as compression algorithm for backward compatibility.
```

The term ```gzip``` is  more widely used than ```GNU zip```